start/software/software-transfer-via-htcondor-or-http.md

```diff
@@ -2,94 +2,60 @@
-	Ariel      |43 (0.42%) ++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Aubrey     |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Baryshnikov|42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Bahia      |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Angstrom   |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Asoka      |42 (0.41%) +++++++++++++++++++++++++++++++++++++++++++++++++++++
-	Alcatraz   |41 (0.40%) ++++++++++++++++++++++++++++++++++++++++++++++++++++
+For some OSG Connect users, it may be necessary or advatageous to install your own software. This guide will 
+describe steps and important considerations for transferring your installed software via the HTCondor submit file.  
 
+## Important Considerations
+
+How you transfer your software via HTCondor will depend on the size and location of the executable binary or tar archive 
+that needs to be transferred along with your jobs. Only software files <100MB staged in your `/home` directory or 
+files >100MB but <1GB staged in your `/public` directory should be transferred via HTCondor using the steps described below. Please see [Introduction to Data Management on OSG Connect](https://support.opensciencegrid.org/support/solutions/articles/12000002985) for more information regarding our policies for staging files (data, software, etc.) in OSG Connect.  
+
+Keep in mind that HTCondor will always transfer the script or binary that is specified as the `executable` in your submit 
+file. Thus, it is not necessary to follow the steps below if your software is the designated executable in your submit file.  
+
+The steps and considerations described below also apply to transferring data, and other files, as described in the [Transfering Data With HTCondor](https://support.opensciencegrid.org/support/solutions/articles/5000639787) guide.
+
+## Transferring Software From `/home`
+
+Software executables and tar archives that are <100MB should be staged in your `/home` directory. To transfer software 
+from your `/home` directory use the `transfer_input_files` statement in your HTCondor submit file. For example:
+
+	transfer_input_files = path/to/my_software.tar.gz
+
+> When using `transfer_input_files` to transfer files located in `/home`, keep in mind that the path to the file is 
+> relative to the location of the submit file. If you have software located in a different `/home` subdirectory,
+> we recommend specifying the full path those files, which is also a matter of good practice, for exmaple:
+> ```
+> transfer_input_files = /home/username/path/to/my_software.tar.gz
+> ```
+> Where `username` refers to your OSG Connect username.
+
+In addition to any software needed for your jobs, be sure to also include other required input files, for example:
+
+	transfer_input_files = path/to/my_software.tar.gz, path/to/my_input.csv
+
+## Transferring Software from `/public`

```

Comment:
from `/public` or the internet?

Reply:
yeah, I wasn't sure what to use for the header b/c of the blast precompiled  binary example, maybe make another section with header for "... From the Internet" and then can also include the jar transfer example?

Reply:
On further thought, I would just stick w/ `/public` and either just mention that it also works for other links, or don't cover that at all.